core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -104,12 +107,12 @@ var ExplorationEditorFeedbackTab = function() {
     var rowCount = await rows.count();
     for (var i = 0; i < rowCount; i++) {
       var row = await rows.get(i);
-      await row.click();
+      await action.click('Row', row);
```

Comment:
Could you make this message more descriptive? It would be helpful to include that this is a suggestionRow and the row index